app/controllers/orgs/rosters_controller.rb

```diff
@@ -146,6 +175,60 @@ def download_roster
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/AbcSize
 
+    def select_google_classroom
+      @google_classroom_courses = fetch_all_google_classrooms
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def search_google_classroom
+      courses_found = fetch_all_google_classrooms.select do |course|
+        course.name.downcase.include? params[:query].downcase
+      end
+
+      respond_to do |format|
+        format.html do
+          render partial: "orgs/rosters/google_classroom_collection",
+                 locals: { courses: courses_found }
```

Comment:
Tried to add this but it hits a Linter Error for AlignHash https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_layout.md#layoutalignhash.

I think it makes sense to update the rubocop rule, didn't want to turn it off for everything because we still want the check but I'm sure there's some configuration we can find in a future PR.